cdc/model/sink.go

```diff
@@ -332,11 +339,33 @@ type RowChangedEvent struct {
 	ReplicatingTs Ts `json:"-" msg:"-"`
 }
 
+// RowChangedEvents represents a set of events
+type RowChangedEvents []*RowChangedEvent
+
+// Len is the number of elements in the collection.
+func (e RowChangedEvents) Len() int {
+	return len(e)
+}
+
+func (e RowChangedEvents) Less(i, j int) bool {
+	return len(e[i].Columns)-len(e[i].PreColumns) <
```

Comment:
This is not easy to understand directly, please add some comments about how the condition comes up.

Reply:
changed to another implementation